doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py

```diff
@@ -230,6 +230,7 @@ def plan_cartesian_path(self, scale=1):
     ## ^^^^^^^^^^^^^^^
     ## You can plan a Cartesian path directly by specifying a list of waypoints
     ## for the end-effector to go through:
+    ## (If executing  interactively in a Python shell, set scale = 1.0).
```

Comment:
No need for parenthesis here, just replace ":" with "."

Reply:
Updated.